### PR TITLE
feat: update population data based on indicator data

### LINKED ISSUE #553: Propagate updated population figures from indicators into www-covidcast
**Actual Behavior:**

Population in NYC boroughs is set to 1 in the map due to an old exception in JHU handling. We now use the combo source for cases & deaths, which uses USAFacts for NYC, and does not require this exception; JHU has additionally updated their geo policies and doesn't need the exception either.

**Steps to Reproduce the Bug:**

1. go to https://covidcast.cmu.edu/index.html?sensor=indicator-combination-confirmed_7dav_incidence_num&level=county&date=20201006&signalType=value&encoding=color&mode=overview&region=36081
2. Mouse over Queens County

**Expected behavior**

Population: 2253858

**Screenshots**

image
image

**Rating Scale (1 is minor, 7 is severe):** 2? it's wrong, but the only place it shows up is the mouseover -- the prop figures come from the API, so they aren't affected.

**Additional context**

(used by combo cases/deaths for most locations) Population file used by USAFacts: https://github.com/cmu-delphi/covidcast-indicators/blob/main/usafacts/static/fips_population.csv

(used by combo cases/deaths for Puerto Rico) Population file used by JHU:https://github.com/cmu-delphi/covidcast-indicators/commit/6fda1b21a227dcda773f43b448da6060ba161bf7

Population file used by map (counties): https://github.com/cmu-delphi/www-covidcast/blob/main/src/maps/processed/county.csv

### PR DESCRIPTION
**Pull request by sgratzl**
closes #553

**Prerequisites**:

- [x] Unless it is a hotfix it should be merged against the `dev` branch
- [x] Branch is up-to-date with the branch to be merged with, i.e. `dev`
- [x] Build is successful
- [x] Code is cleaned up and formatted

### Summary

updates the population with the latest file as mentioned in the issue when processing the raw data:

image

### CODE REVIEW

File: src/maps/process.js [hunk 1]
```diff
@@ -110,7 +112,22 @@ function msa(level = 'msa') {
   return geo;
 }
 
-function counties(level = 'county') {
+async function generatePopulationLookup() {
+  const csv = csvParse(
+    await fetch(
+      'https://raw.githubusercontent.com/cmu-delphi/covidcast-indicators/main/_delphi_utils_python/delphi_utils/data/fips_pop.csv',
```
Comment (tildechris) [inline src/maps/process.js:118]: Are there any cross-site considerations to be made when pulling this data from github directly?  Would it be better to pull the data down and package it as part of the site at build time?

**Review by tildechris — CHANGES_REQUESTED** [review]
Approach LGTM, but should we incorporate fips_pop.csv as a site resource?

File: src/maps/process.js (hunk 1 above)
Comment (sgratzl) [reply src/maps/process.js:118]: the raw data is just processed on demand. It will generate among others the CSV file that is also checked in.

Comment (tildechris) [reply src/maps/process.js:118]: Ah so, this is only at build time?  That makes sense.  It'd be nice if there was a npm package that we could pull down as a first-class dependency, but until then this make sense.

**Review by tildechris — APPROVED** [review]
LGTM.

**Pull request merged**